test/ngResource/resourceSpec.js

```diff
@@ -1356,5 +1357,36 @@ describe('resource', function() {
       );
   });
 
+  it('If timeout promise is resolved, cancel the request', function() {
```

Comment:
nit: can you change the test description to match the grammar of other ones in the file?